beacon-chain/node/config.go

```diff
@@ -77,6 +79,30 @@ func configureBuilderCircuitBreaker(cliCtx *cli.Context) error {
 	return nil
 }
 
+// configureBlobRetentionEpoch sets the for blob retention based on command-line context. It overrides the existing network configuration
+// for blob retention if the input epoch is greater than or equal to the spec default value.
+// An error if the input epoch is smaller than the spec default value.
+func configureBlobRetentionEpoch(cliCtx *cli.Context) error {
+	// Check if the blob retention epoch flag is set.
+	if cliCtx.IsSet(flags.BlobRetentionEpoch.Name) {
+		// Retrieve and cast the epoch value.
+		epochValue := cliCtx.Uint64(flags.BlobRetentionEpoch.Name)
+		e := primitives.Epoch(epochValue)
+
+		// Validate the epoch value against the spec default.
+		if e < blobRetentionEpochSpecValue {
+			return fmt.Errorf("input blob retention epoch smaller than spec default, %d < %d", e, blobRetentionEpochSpecValue)
+		}
+
+		// Copy the existing beacon network configuration.
+		c := params.BeaconNetworkConfig().Copy()
+		c.MinEpochsForBlobsSidecarsRequest = e
```

Comment:
I don't like modifying this variable. We use it for instance in p2p RPC code. I think it's going to be less likely to cause problems if we identify places where the user should be able to specify a different period - which I think would probably be limited to the db package? - and make a new variable that defaults to the spec value, with the option of the user overriding it.